MIGRATION.md

```diff
@@ -230,9 +233,27 @@ This was done to remove a hard dependency on `https-proxy-agent` when running
    authorization: 'authorization'
  }`
 
-### 8. Bug fix: Setting an explicit `algorithm` when generating a wallet works now
+### 8. `Wallet` functions default to `ed25519` instead of `secp256k1` signing algorithm
+
+With 3.0, we updated the default signing algorithm used by the Wallet object from secp256k1 to ed25519 in order to default to the higher-performance algorithm. This is a breaking change to all functions used to generate a Wallet, so if you have a pre-existing XRPL account that you're using to generate a specific Wallet using older versions of xrpl.js, you may need to specify that you are using secp256k1 as the algorithm to decode your private key / seed / etc to get the same behavior as before. See below for specifically how to update your code.
+
+If you are creating new accounts each time (ex. via `Client.fundWallet` or `Wallet.generate`), you do not need to specify the signing algorithm.
 
-`Wallet.generate()` and `Wallet.fromSeed` were ignoring the `algorithm` parameter. This means that if you were manually specifying `algorithm` in any `Wallet` constructors, you may generate a different `Wallet` keypair when upgrading to 3.0. In that case to get the same generated wallets as before, don’t specify the `algorithm` parameter.
```

Comment:
Some form of this text should still be included in this section, to explain why the change happened. Otherwise it seems random.

Reply:
okay, updated in [a16773e](https://github.com/XRPLF/xrpl.js/pull/2659/commits/a16773ee21847ed903fca9f0c6b77098890201fb)